doc/source/user_guide/getting_started.rst

```diff
@@ -41,3 +41,18 @@ from image files, using :func:`skimage.io.imread`: ::
     >>> from skimage import io
     >>> moon = io.imread(filename)
 
+Use natsort to load multiple images: ::
```

Comment:
```suggestion
Use `natsort <https://pypi.org/project/natsort/>`_ to load multiple images ::
```

Reply:
@soupault Thanks for input. I didn't know to do this, so this is a big help. Thank you, and I'll be commit-ing your change.

Reply:
@D-Bhatta I have looked it up myself in the same file :sweat_smile:. It is difficult to remember many things at once.